src/subscriber.ts

```diff
@@ -380,6 +509,8 @@ export interface SubscriberOptions {
   flowControl?: FlowControlOptions;
   useLegacyFlowControl?: boolean;
   streamingOptions?: MessageStreamOptions;
+
+  /** @deprecated Unset and use context propagation. */
```

Comment:
Yeah, this could be expanded. Basically in the previous iteration of support, you had to set a flag explicitly to enable tracing. In the new one, that flag should only be set if you want the old attributes and measuring methods and such. So the intent here is "unset this flag in the future, and just instantiate a tracing provider so that it uses standard context propagation". The actual wording doesn't need to be that complex, though.